eventmesh-examples/src/main/java/org/apache/eventmesh/grpc/sub/app/service/SubService.java

```diff
@@ -115,8 +115,8 @@ public void cleanup() {
      * Count the message already consumed
      */
     public void consumeMessage(String msg) {
-        logger.info("consume message {}", msg);
+        logger.info("consume message: {}", msg);
         countDownLatch.countDown();
-        logger.info("remaining number of messages to be consumed {}", countDownLatch.getCount());
+        logger.info("remaining number of messages to be consumed : {}", countDownLatch.getCount());
```

Comment:
For  code style consistency, you can delete this space.
```suggestion
        logger.info("remaining number of messages to be consumed: {}", countDownLatch.getCount());
```